contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+- **Consider marketing and product alignment:** Sometimes poor retention comes from acquiring the wrong users. This could be a marketing problem, or you might have the wrong ideal customer profile.
+
+- **Learn about your power users:** It's great to have power users, but they're also more sensitive to product changes. It's important to identify who they are, what they do, and what they value so you can avoid retention problems. See: [How to identify and analyze power users in PostHog](/tutorials/power-users).
+
+- **Build great new features:** If you're talking to your users, and learning about your power users, then you should be well-placed to build awesome new features your existing (and potential) customers will love. I'm told this is "creating value" in the official business lingo.
+
+- **Create a customer success team:** Customer success > outbound sales a lot of the time. [Our customer success team](/handbook/small-teams/customer-success) owns all inbound contact requests, assisting new users and introducing them to the right PostHog people at the right time. An effective CS team will help onboard new customers and retain them long term.   
+
+- **Buy or copy the new hotness:** Aka, doing a Zuckerberg. This probably isn't a viable option for most companies, but YOLO right? (Yes, I did include this one mostly for the lols. It's not serious advice, probably.)
+
+### The things you need to improve retention
+
+You're gonna need a few things to track, analyze and improve customer churn, such as:
+
+1. The ability to identify your new/current users (e.g. job role, location etc.)
+
+1. An analytics tool to help you track user churn, and analyze use behavior.
+
+1. Session recording so you can watch real interactions with your product.
+
+1. A way to test new features and improvements to validate if they work.
+
+1. The ability to safely roll out new features to test groups and specific cohorts
+
+1. Product-minded engineers or product managers who talk to users and ship fast
+
+Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.
+
+And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.
+
+Check out [our product page](/product) if this sounds interesting to you.
+
+/pitch over
+
+## Takeaways on churn analysis
+
+Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.
+
+To conclude, here are five takeaways you can use to sound smart in meetings, or perhaps share on LinkedIn 😱 with a hashtag – I won't judge, not publicly at least:
```

Comment:
Maybe too far, but at least if you take out some of the earlier snark, this will be funny